src/videojs.ima.js

```diff
@@ -428,6 +428,7 @@
         fullscreenDiv.className = 'ima-fullscreen';
         adContainerDiv.style.width = window.screen.width + 'px';
         adContainerDiv.style.height = window.screen.height + 'px';
+        adContainerDiv.className = 'ima-ad-container--fullscreen';
```

Comment:
Sorry for my tardiness, was using a bem style class naming, but as it's not being used elsewhere in the codebase, sure, single dash it is